qiskit/visualization/tools/pi_check.py

```diff
@@ -44,13 +44,20 @@ def pi_check(inpt, eps=1e-6, output='text', ndigits=5):
     Raises:
         QiskitError: if output is not a valid option.
     """
+    complex_check = 'False'
+
     if isinstance(inpt, ParameterExpression):
         return str(inpt)
     elif isinstance(inpt, str):
         return inpt
+    elif isinstance(inpt, complex) and inpt.real == 0:
```

Comment:
why `inpt.real == 0`?

Reply:
I think I had an incorrect logic in mind that real numbers were being handled by the regular float. This code worked only in scenarios where either of real and imaginary is non-zero. I have now modified the code to take into account both real and imaginary as non-zero values.